x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ssl/StoreKeyConfig.java

```diff
@@ -78,6 +78,18 @@ X509ExtendedKeyManager createKeyManager(@Nullable Environment environment) {
         Path ksPath = keyStorePath == null ? null : CertParsingUtils.resolvePath(keyStorePath, environment);
         try {
             KeyStore ks = getStore(ksPath, keyStoreType, keyStorePassword);
+            // TBD: filte out only http.ssl.keystore
+            ArrayList<String> aliases = Collections.list(ks.aliases());
+            if (this instanceof StoreKeyConfig && aliases.size() > 1) {
```

Comment:
Do we need `this instanceof StoreKeyConfig` ? We're in `StoreKeyConfig` right ?

Reply:
Right, good catch, leftover from a previous iteration. Removed.